Modules/Loadable/Markups/MRML/vtkMRMLMarkupsROINode.h

```diff
@@ -240,6 +240,11 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsROINode : public vtkM
   /// Get the implicit function that represents the ROI in world coordinates.
   vtkGetObjectMacro(ImplicitFunctionWorld, vtkImplicitFunction);
 
+  /// Create ROI box as surface mesh in the world coordinate system as a new vtkPolyData object.
+  /// In C++, the owner must take ownership of the returned object (delete when no longer used).
+  /// In Python, the returned object automatically takes ownership of the returned object (no need to manually delete).
```

Comment:
To be specific the ownership is now correctly transferred to python, so conceptually Python is responsible for "deleting" the object.